easybuild/easyconfigs/g/GCTA/GCTA-1.92.2beta.eb

```diff
@@ -0,0 +1,31 @@
+# Easybuild easyconfig
+#
+# John Dey jfdey@fredhutch.org fizwit@github
+# Notes: Could build from source if MKL libraries/include files are available
+# change Makefile by removing 'static', requires MKL libraries, requires newer version of GLIBCXX_3.4.x
```

Comment:
@fizwit If changes must be applied to the `Makefile`, adding a patch is the best way.